### PR TITLE
added link to nightly builds in the apopriate place of RM

### PR DESCRIPTION
**Pull request by jeditobe**
## Purpose

added link to nightly builds in the apopriate place for convenience

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,8 @@ build directory. This will create a CD image with a filename `bootcd.iso`.
 
 See ["Building ReactOS"](http://www.reactos.org/wiki/Building_ReactOS) for more details.
 
+You can always download fresh binary builds of bootable images from the ["special page"](https://www.reactos.org/getbuilds/)
+
```
Comment (Fraizeraust) [inline README.md]: I'd argue `Daily Builds` (or anything similar) is much better than "special page".

Comment (jeditobe) [reply README.md]: fixed

**Comment by cernodile** [comment]
Can you please squash your commits? That's the only problem I see here.

**Comment by jeditobe** [comment]
I can only recreate, because i use web-editor

**Comment by HBelusca** [comment]
Then please take the time to learn and use something else than the web-editor (hint: making development/PRs from a phone is a very bad idea!)

**Pull request closed without merging**